examples/pycharm-plugin/build.gradle

```diff
@@ -0,0 +1,5 @@
+intellij {
+    // PC-Snapshot is also available here
```

Comment:
really? do we publish community snapshots publicly?

Reply:
yeap, we want to publish pycharm community as well